python/plugins/db_manager/db_plugins/postgis/connector.py

```diff
@@ -27,7 +27,14 @@
 from functools import cmp_to_key
 
 from qgis.PyQt.QtCore import QRegExp, QFile, QCoreApplication
-from qgis.core import Qgis, QgsCredentials, QgsDataSourceUri, QgsCoordinateReferenceSystem
+from qgis.core import (
+    Qgis,
+    QgsCredentials,
+    QgsDataSourceUri,
+    QgsProviderRegistry,
+    QgsAbstractDatabaseProviderConnection,
```

Comment:
Do you need to import this? It's abstract, you get the concrete instance out of the factory.

Reply:
I removed the import completely, with c01e34bf12